portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,26 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+function audioPlayer() {
+  console.log('Running audio player.');
+  currentSong = 0;
+  player = document.getElementById('audio-player');
+  songs = document.querySelectorAll('#song-list li a');
+  console.log('Adding Event Listener to song links...');
+  songs.forEach((song, index, songList) => {
+    song.addEventListener('click', (event) => {
+      console.log('Clicked song.');
+      event.preventDefault();
+      songList[currentSong].parentNode.className = '';
```

Comment:
Just curious, why is this required?

Reply:
This was so that when I change the song, the previous song no longer had a class name of current song and I was planning on styling it such that the current song was in a different colour than the others but I hadn't gotten to that yet. I should have made more comments clarifying this

Reply:
Okay, thanks. One suggestion would be to avoid adding changes that are unused currently - like you could add the 'current-song' class in the pull request where you are also adding the styling, so that it makes more sense :).